src/derivatives/imaging.md

```diff
@@ -293,9 +293,11 @@ In this case, the mask suffix MUST be used,
 the [`label` entity](../appendices/entities.md#label)) SHOULD be used
 to specify the masked structure
 (see [Common image-derived labels](#common-image-derived-labels)),
-and the [`seg` entity](../appendices/entities.md#segmentation) SHOULD be defined.
+and the [`seg` entity](../appendices/entities.md#segmentation) specifying
+what segmentation generated the mask SHOULD be defined.
```

Comment:
This is grammatically not clear to us. Can you update the phrasing please?

Reply:
I'm right now working on this. Besides the grammar, I'm realizing this is a bit messy in the current specs (cc @effigies) in that this paragraphs is specifically talking about `_mask` although the corresponding subsection is above (and we don't have `seg-` defined for `_mask` which is interesting).

Reply:
It gets better, the discrete segmentation template doesn't define `label-<label>`!

Reply:
I'm taking a stab at it and then will ask for your feedback :)

Reply:
@effigies https://github.com/bids-standard/bids-specification/pull/1856/commits/44cb808ba39bdef152a748d7c5b92f2f1faa1ac5

Reply:
Please note that probabilistic segmentations are probably in worse shape.

Reply:
This reads fine to me.

Reply:
Since the former weird structure is no longer proposed and the changes are more substantial than reformulating a sentence, I'll resolve this conversation and wait for @melanieganz's review of the whole commit changes (https://github.com/bids-standard/bids-specification/commit/44cb808ba39bdef152a748d7c5b92f2f1faa1ac5).